### DISCUSSION TITLE
[Feature Request] Forks insights more info about fork

### DISCUSSION THREAD

**Discussion by markrity** [community/community#23763] [New to GitHub]
<p>It would be awesome to know more info about forks, to know how many commits they are behind and how many above.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/9a6d960eb5b5056b2b99f4201abe94cd630e4641.png" data-download-href="/uploads/short-url/m28jHjaxrx4PGRnpCgkqasA1qj7.png?dl=1" title="Screen Shot 2020-01-05 at 18.02.07.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11563691/181079251-50bfe6a9-987f-491c-8759-3db30bc77c35.png" alt="Screen Shot 2020-01-05 at 18.02.07.png" data-base62-sha1="m28jHjaxrx4PGRnpCgkqasA1qj7" width="690" height="292" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/9a6d960eb5b5056b2b99f4201abe94cd630e4641_2_690x292.png, https://aws1.discourse-cdn.com/github/optimized/1X/9a6d960eb5b5056b2b99f4201abe94cd630e4641_2_1035x438.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/9a6d960eb5b5056b2b99f4201abe94cd630e4641_2_1380x584.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/9a6d960eb5b5056b2b99f4201abe94cd630e4641_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-01-05 at 18.02.07.png</span><span class="informations">1884×800 120 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p><strong>How it helps ? </strong> When I’m looking at non maintained or repo with some known issues, I look at forks if anyone has fixed that issue, but I have no idea from insights as they look right now, if they have commited anything new at their repo,</p>
<p>So I need to go 1 after 1 , each repo…</p>

**Comment by BelHFL** [comment, marked as answer]
<p>Hi <a class="mention" href="/markrity">@markrity</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>

**Comment by oriadam** [comment]
<p><strong>last commit date</strong> would be helpful as well. i will immediately know which forks are abandoned and ignore them.<br>
<strong>date of last merge from base</strong> would be also helpful for the same reason.<br>
alternatively, add a filter of <strong>show only active forks</strong> or similar.</p>
<p>although it may cost more in terms of db processing, it will make the insights/forks page useful, and save the aimless browsing through forks which cost both cpu, bandwidth and user time.</p>
<p>thank you for adding it to your internal feature list, and keep up the good work :star_struck:</p>
